docs/en/latest/plugins/redirect.md

```diff
@@ -34,7 +34,7 @@ The `redirect` Plugin can be used to configure redirects.
 
 | Name                | Type          | Required | Default | Valid values | Description                                                                                                                                                                                                                                                                                                                                                                        |
 |---------------------|---------------|----------|---------|--------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| http_to_https       | boolean       | False    | false   |              | When set to `true` and the request is HTTP, it will be redirected to HTTPS with the same URI with a 301 status code.                                                                                                                                                                                                                                                               |
+| http_to_https       | boolean       | False    | false   |              | When set to `true` and the request is HTTP, it will be redirected to HTTPS with the same URI with a 301 status code, contains the query string.                                                                                                                                                                                                                                                               |
```

Comment:
```suggestion
| http_to_https       | boolean       | False    | false   |              | When set to `true` and the request is HTTP, it will be redirected to HTTPS with the same URI with a 301 status code.  Note the querystring from the raw URI will also be contained in the Location header.                                                                                                                                                                                                                                                             |
```

Reply:
Done